CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
-# 6.6.0 (unreleased)
+# 6.6.0 (2020-01-27)
```

Comment:
Would add it under a new `# 6.7.0 (unreleased)` section instead since introduced after branching of 6.6.x release branch. Either keep notice or add it under breaking changes section.